### PR TITLE
Updated two dependencies for gentoo.

### PR DESCRIPTION
**Pull request by allenh1**
There were two small problems with the package selection. This should satisfy everything.

### CODE REVIEW

File: rosdep/base.yaml [hunk 1]
```diff
@@ -2708,7 +2708,7 @@ pkg-config:
   debian: [pkg-config]
   fedora: [pkgconfig]
   freebsd: [pkg-config]
-  gentoo: [dev-util/pkgconfig]
+  gentoo: [dev-util/pkgconf]
```
Comment (dirk-thomas) [inline rosdep/base.yaml]: Both seems to exist:
* https://packages.gentoo.org/package/dev-util/pkgconfig
* https://packages.gentoo.org/package/dev-util/pkgconf

Why is the new one favored over the previous one?

**Comment by allenh1** [comment]
Most gentoo applications use dev-util/pkgconf, and both cannot be on the system at the same time.

**Comment by dirk-thomas** [comment]
It sounds like a big change for every current user to switch?

I will keep this for @tfoote to merge.

**Comment by tfoote** [comment]
What are the implications for someone who's already using pkgconfig?

**Comment by allenh1** [comment]
pkgconf is an identical system to pkgconfig. The change should not affect anyone. I should also point out that my system won't even let me install pkgconfig. I did a clean install to make sure this wasn't anything I had done abnormally and pkgconf was pulled in with the install of Gnome. I suppose the safest thing would be to allow either, but I don't know how to do that or if it's even a possibility.

**Comment by tfoote** [comment]
If it's incompatible as it stands with a fresh install with gnome that seems reasonable to switch.

**Pull request merged**